config/versions/versions.go

```diff
@@ -30,7 +30,6 @@ import (
 const LatestVersion = -1
 
 var (
-	errMissingVersion      = errors.New("missing version")
 	errVersionIncompatible = errors.New("version does not implement ConfigVersion or ExchangeVersion")
 	errModifyingExchange   = errors.New("error modifying exchange config")
 	errNoVersions          = errors.New("error retrieving latest config version: No config versions are registered")
```

Comment:
Part of me dislikes that this _lying_ output that I upgraded v4 when `v4.go` does not exist. But it is a requiresDependency PR and as such, any config version upgrade requires a review to merge and so it is on them
```
go run . upgrade --edit
GoCryptoTrader: config-helper tool
2025/03/17 12:06:40 Running upgrade to config version 4
2025/03/17 12:06:40 Running upgrade to config version 5
```
You could check if one `cPatch, ok := patch.(ConfigVersion); ok `/ `ePatch, ok := patch.(ExchangeVersion); ok` passes before printing, but I think its being a bit much for me grumbling

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/b63781f

Reply:
It looked resolved, thanks! However, the check fails:
<img width="663" alt="image" src="https://github.com/user-attachments/assets/951767a7-f3f8-4004-9e47-0841d0085281" />
This appears to be the can't have nil for interface trap you've always tried to warn us about